packages/framework-extra-bundle/DependencyInjection/Integration/CronJobIntegration.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Bundle\FrameworkExtraBundle\DependencyInjection\Integration;
+
+use Draw\Component\Console\Command\PurgeExecutionCommand;
+use Draw\Component\Console\Entity\Execution;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+class CronJobIntegration implements IntegrationInterface, PrependIntegrationInterface
```

Comment:
FYI I am think about moving this class to the package itself but I am not sure yet. Continue like this right now and we might refactor all the packages in a couple of month.

Better to have only one way of doing things right now.